cloudfront/invalidation/lambda/src/cache_invalidation.test.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@jest/globals';
+import { SNSEvent, SNSEventRecord } from 'aws-lambda/trigger/sns';
+import { Context } from 'aws-lambda';
+import * as AWS from "aws-sdk";
+import { CreateInvalidationRequest as CloudFrontInvalidationRequest } from 'aws-sdk/clients/cloudfront';
+import * as AWSMock from "aws-sdk-mock";
+
+import { handler } from './cache_invalidation';
+
+test('test request', async () => {
+  const message = {
+    paths: ["foo", "bar"],
+    reference: "my-test-reference"
+  };
+  const fakeEvent = getFakeEvent(JSON.stringify(message));
+
+  AWSMock.setSDKInstance(AWS);
+  let called = false;
+  AWSMock.mock("CloudFront", "createInvalidation", (params: CloudFrontInvalidationRequest, callback: Function) => {
+    called = true;
+    callback(null, null);
+  })
+
+  await handler(fakeEvent, {} as Context, () => {});
+  expect(called).toBe(true);
```

Comment:
Should we be restoring? `AWSMock.restore('CloudFront');`

Reply:
Yes we should! Updated.